popup.html

```diff
@@ -119,17 +119,6 @@
         </div>
         <div class="ml-auto">
             <div class="row">
-                <div class="col px-1">
-                    <a href="mailto:info@fieldteam6.org?subject=Feedback%20on%20BYOP" class="text-secondary text-center" target="_blank" title="Have feedback?">
-                        <div class="miniIcon">
-                            <svg class="w-6 h-6" fill="none" stroke="currentColor" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-                                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2"
-                                    d="M9.663 17h4.673M12 3v1m6.364 1.636l-.707.707M21 12h-1M4 12H3m3.343-5.657l-.707-.707m2.828 9.9a5 5 0 117.072 0l-.548.547A3.374 3.374 0 0014 18.469V19a2 2 0 11-4 0v-.531c0-.895-.356-1.754-.988-2.386l-.548-.547z">
-                                </path>
-                            </svg>
-                        </div>
-                    </a>
-                </div>
```

Comment:
we already have a help link that people can use to email us.  we probably don't need another.